custom/docker/build.gradle

```diff
@@ -111,5 +95,3 @@ jib {
         }
     }
 }
-
-// tasks.jibDockerBuild.dependsOn(':fineract-provider:bootJar')
```

Comment:
are you sure this need to be commented out?

Reply:
well... this is the latest... if no one complained until now then I assume it's not needed...

Reply:
mea culpa. i missed you deleted the commented out code piece only 😅